pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -986,7 +986,7 @@ func (og *operationGenerator) GenerateMapVolumeFunc(
 		volume.VolumeOptions{})
 	if newMapperErr != nil {
 		eventErr, detailedErr := volumeToMount.GenerateError("MapVolume.NewBlockVolumeMapper initialization failed", newMapperErr)
-		og.recorder.Eventf(volumeToMount.Pod, v1.EventTypeWarning, kevents.FailedMapVolume, eventErr.Error())
+		og.recorder.Eventf(volumeToMount.Pod, nil, v1.EventTypeWarning, kevents.FailedMapVolume, "Mounting Volume", eventErr.Error())
```

Comment:
I think we should mark this as "Mapping Volume" to be consistent with the kubelet event type (and also to be consistent with the string we provide below on line 1167 for the same kubelet event type).